keyboards/melody96/config.h

```diff
@@ -3,8 +3,8 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6060
+#define VENDOR_ID       0x594d // "YM"
+#define PRODUCT_ID      0x0001
```

Comment:
```suggestion
#define PRODUCT_ID      0x0096
```
?

Reply:
Issues with this,  you're assuming there's only one product named 96 from YMKD. That isn't the case though, YMKD has more than one thing that has 96 in the product name such as the [YMD96](https://github.com/qmk/qmk_firmware/tree/master/keyboards/ymd96).

So giving this product the `0x0096` as the product ID, wouldn't work as a naming conversion for other products as then you have duplicates of products that *should* use `0x0096` as its product ID.

Reply:
Fair enough. How about `0x4D96` for ASCII "M" + 96?

Reply:
That would be a better idea